e2e/src/web/specs/shared-link.e2e-spec.ts

```diff
@@ -40,6 +40,16 @@ test.describe('Shared Links', () => {
       },
       { headers: asBearerAuth(admin.accessToken) }
     );
+    sharedLinkPassword = await createSharedLink(
```

Comment:
I think there is an apiUtils method that is more concise you could use.

Reply:
There is indeed, I'll update it